sim/mage/arcane_surge.go

```diff
@@ -0,0 +1,80 @@
+package mage
+
+import (
+	"time"
+
+	"github.com/wowsims/sod/sim/core"
+	"github.com/wowsims/sod/sim/core/proto"
+)
+
+func (mage *Mage) registerArcaneSurgeSpell() {
+	if !mage.HasRune(proto.MageRune_RuneLegsArcaneSurge) {
+		return
+	}
+
+	level := float64(mage.GetCharacter().Level)
+	actionID := core.ActionID{SpellID: int32(proto.MageRune_RuneLegsArcaneSurge)}
+	baseCalc := (13.828124 + 0.018012*level + 0.044141*level*level)
+	baseDamageLow := baseCalc * 2.26
+	baseDamageHigh := baseCalc * 2.64
+	spellCoeff := .429
+	cooldown := time.Minute * 2
+	auraDuration := time.Second * 8
+
+	manaMetrics := mage.NewManaMetrics(actionID)
```

Comment:
You should be able to pass the ArcaneSurge spells own mana metrics instead of this